salt/orch/removal.sls

```diff
@@ -6,6 +6,8 @@
 {%- set replacement = salt['pillar.get']('replacement', '') %}
 {%- if replacement %}
   {%- set replacement_provided = True %}
+{%- else %}
+  {%- set replacement_provided = False %}
```

Comment:
Maybe Jinja would accept `set replacement_provided = (replacement != '')`

Reply:
I didn't try that one specifically, will try.